setup.py

```diff
@@ -33,7 +33,11 @@
         'tenacity==4.10.0',
         'setuptools_scm==2.0.0'
     ],
-    scripts=['cromwell_tools/scripts/cromwell-tools'],
+    entry_points={
+        'console_scripts': [
+            'cromwell-tools = cromwell_tools._cli:main'
```

Comment:
I'd prefer to name this something other than _cli. Maybe it should be cromwell_tools.py or main.py, so it's obvious from the file name which one is the entrypoint.